### PR TITLE
sgx: stop using local source hooks for EPC registration

### PR DESCRIPTION
**Pull request by mythi**
Fixes: #1380

### CODE REVIEW

File: cmd/operator/README.md [hunk 1]
```diff
@@ -26,11 +26,9 @@ The default operator deployment depends on NFD and cert-manager. Those component
 Install NFD (if it's not already installed) and node labelling rules (requires NFD v0.10+):
```
Comment (tkatila) [inline cmd/operator/README.md:26]: As this now depends on a newer NFD, it would be good to mention the version here?

**Review by tkatila — COMMENTED** [review]
Should we keep building the initcontainer?

File: cmd/operator/README.md (hunk 1 above)
Comment (mythi) [reply cmd/operator/README.md:26]: yes makes sense

**Comment by mythi** [comment]
> Should we keep building the initcontainer?

initcontainer can also be used without NFD so that needs to stay. I'm also keeping the existing behaviour in the operator if someone wants it.

File: cmd/operator/README.md (hunk 1 above)
Comment (mythi) [reply cmd/operator/README.md]: fixed

**Review by tkatila — APPROVED** [review]

**Comment by mythi** [comment]
@hj-johannes-lee PTAL and merge if ok

**Review by tkatila — APPROVED** [review]

**Review by hj-johannes-lee — APPROVED** [review]

**Pull request merged**